LICENSE-APACHE

```diff
@@ -1,198 +1,198 @@
-                              Apache License
+                            Apache License
```

Comment:
we'll want to exclude our licenses from editorconfig

Reply:
I didn't see a way to do this without adding a comment at the header of the file. Will take another look tomorrow.